Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM docker.io/alpine:latest
+
+RUN apk add --no-cache git && \
+    mkdir /workdir
```

Comment:
There's no need for `/workdir`

Reply:
On a second thought, I take back my original comment. It's still gonna be useful for users to have a `WORKDIR` set up. Please add it back as `/work`. And I apologize for it.

Reply:
Alright it is done 😉